.github/workflows/document-lint/.markdownlint.jsonc

```diff
@@ -0,0 +1,13 @@
+{
+  // Example markdownlint JSON(C) configuration with all properties set to their default value
+  // https://github.com/DavidAnson/markdownlint/blob/main/schema/.markdownlint.jsonc
+
+  // Default state for all rules
+  "default": false,
+
+  // Path to configuration file to extend
+  "extends": null,
+
+  // MD032/blanks-around-lists - Lists should be surrounded by blank lines
+  "MD032": true
```

Comment:
if we are not checking for all violations right away we should create an epic to collect the rules we intend to roll out. I think this should be part of the epic  https://github.com/DavidAnson/markdownlint/blob/v0.27.0/doc/md031.md

We will also need an epic for fixing violations where team members can volunteer to fix them. We can also look for an autoformatter option.

Reply:
I am not sure if we could benefit from the tool without involve much effort, since there are too much errors which will fail the build, please find the issue #7938 I filed for this

Reply:
This is an optional check, you can see on this very PR that the linter is not going to block PR's merge. 

But if there is an autoformatter instead of a linter  like maybe https://github.com/hukkin/mdformat-gfm that rewrites our files as part of the build and works well even better than just doing the checks